docs/minio-job.md

```diff
@@ -78,16 +78,16 @@ spec:
     - name: add-my-user-1
       op: admin/user/add
       args:
-        user: ${USER}
-        password: ${PASSWORD}
+        user: $(USER)
```

Comment:
`${VAR}` is a bash-ism right?

`$(VAR)` is a bit atypical, or in bash semantics would imply a subshell.

Is this the easiest path, or would changing the logic to support `${VAR}` substitution be harder?

Just curious.